### ISSUE TITLE
embedding Videos from BR Mediathek

### ISSUE THREAD

**Issue by SimonKoehl** [serlo-org/athene2#722]
Can we add BR Mediathek as a source option for our video learning object?

e.g. https://www.br.de/mediathek/video/respekt-10062018-pluralismus-warum-die-demokratie-von-mehr-vielfalt-profitiert-av:5abbbdeb512d0600188593cc

embedding code can be found if you click on "Teilen"

**Comment by inyono** [comment]
Yes, should be possible. Are the videos always online (instead of the timed deletion because legal stuff in ARD + ZDF mediathek)?

TODO:
* [ ] Map `https://www.br.de/mediathek/video/respekt-10062018-pluralismus-warum-die-demokratie-von-mehr-vielfalt-profitiert-av:5abbbdeb512d0600188593cc` to `https://www.br.de/mediathek/embed/respekt-10062018-pluralismus-warum-die-demokratie-von-mehr-vielfalt-profitiert-av:5abbbdeb512d0600188593cc`
* [ ] Use provided embed code with that url (or some similar embed code)

**Comment by br-oliver** [comment]
If you use the provided embed code, you don't have to map, it is ready to use:

<div style="width:100%;padding-top:56.25%;position:relative;background:#000;"><iframe allowfullscreen style="position:absolute;left:0;right:0;bottom:0;top:0;border:0;height:100%;width:100%;" src="https://www.br.de/mediathek/embed/respekt-10062018-pluralismus-warum-die-demokratie-von-mehr-vielfalt-profitiert-av:5abbbdeb512d0600188593cc"></iframe></div>

Unfortunately, the videos are not always online - only for 5 years...

**Comment by inyono** [comment]
@br-oliver:  That's true, but we don't want the user to have to enter that code. So what we are probably going to do is that the user provides the usual video url and we use that entered url and render the embed code.

@SimonKoehl Is the 5-year limit a problem? Resp @br-oliver: do you know if the embedding handles "expired" videos correctly and shows a good error message (Like: "Leider ist dieser Inhalt nicht mehr verfügbar?")? Or can we check via API when the video will expire? Then it wouldn't be such a big problem IMHO if we only use BR videos to enrich the content but don't rely on them to be there.

**Comment by inyono** [comment]
The reason I'm asking: It would be a shame if a good learning entity relies on the video to be there since then the content has to be rewritten after at most 5 years (depending on how long the video is already online). Furthermore: @SimonKoehl: What about the license? If it's cc anyway, we might be better off downloading the video and reuploading it to e.g. Wikimedia Commons (the license would allow that). If not, we have a slight license problem and need to indicate the different license in the content (which we don't do currently I suppose since we only show the license of the "outer" entity and not the embedded one).

**Comment by Knorrke** [comment]
About license: embeded videos also show the license

**Comment by br-oliver** [comment]
@inyono As for the error message: yes. But the licence is not CC, so you would have to use the embedding method to display the content-  then you are legally fine.

The API (in construction) will deliver the date the video expires - and, fyi,  the laws that force public stations to take down their content are in discussion and about to change - in favour of longer availability of videos - we will know details next year.

**Comment by inyono** [comment]
@Knorrke Nice, didn't know.

Okay, everything's fine by me, then 👍

**Comment by SimonKoehl** [comment]
ok, so let's get started with the current circumstances and hope for better laws regarding the expire date and licences in public broadcasting in the future

**Comment by Knorrke** [comment]
@SimonKoehl can you create the license for those videos?

**Comment by SimonKoehl** [comment]
@Knorrke done https://de.serlo.org/license/manage
